src/style.css

```diff
@@ -0,0 +1,14 @@
+.box{
+height: 100px;
+}
+
+
+.red-background{
+  background-color: red;
+}
+.green-background{
+  background-color: green;
+}
+.blue-background{
+  background-color: blue;
+}
```

Comment:
check indentations and code formatting, run linter, add new line in the end of the file